tests/test__mongomock.py

```diff
@@ -1704,6 +1704,14 @@ def test__aggregate20(self):
         ]
         self.cmp.compare.aggregate(pipeline)
 
+    def test__aggregate20b(self):
+        pipeline = [
+            {'$project': {'_id': 0, 'max': {'$max': [5, 9, 1]},
```

Comment:
There are no tests including references to values like `$a`, or `$b`. Either add an extra test for it, or cover this case here.